doc/src/quickstart/index.rst

```diff
@@ -370,7 +276,7 @@ The resulting command is:
 
     > $VTR_ROOT/vpr/vpr \
         $VTR_ROOT/vtr_flow/arch/timing/EArch.xml \
-        blink --circuit_file blink.pre-vpr.blif \
+        ~/vtr_work/quickstart/blink_manual/temp/blink.parmys.blif \
```

Comment:
Why are you passing the full file path? If someone is following your steps correctly, should they not already be in the `~/vtr_work/quickstart/blink_manual/` folder? If you do want to make it explicit, I would recommend creating a `$WRK_DIR` variable, similar to `$VTR_ROOT` instead of passing the whole path.